docs/ssr.md

```diff
@@ -69,3 +69,21 @@ module.exports = {
   plugins: [...otherGatsbyPlugins, 'gatsby-plugin-emotion']
 }
 ```
+
+## Puppeteer
+
+If you are using Puppeteer to prerender your application, you have to disable emotion's `speedy` option to fully render all needed CSS inside the DOM.
+
+```jsx
+import { sheet } from 'emotion'
+
+import App from './App'
+
+sheet.speedy(false)
+
+ReactDOM.render(<App />, document.getElementById('root'))
+```
+
+> Note:
+>
+> You have to disable this option before any styles are written. Putting it close to your entrypoint (without any other styling) is a good approach.
```

Comment:
I wouldn’t really say good approach since there isn’t really any alternative so it should probably be you have to.